hardware/raspberrypi/bcm2711_bootloader_config.md

```diff
@@ -84,28 +84,90 @@ Default: 0x00000001 (with 3 SD boot retries to match the current bootloader beha
+Default: 0
+Version: stable/pieeprom-2020-01-17.bin  
+
+### TFTP_PREFIX_STR
+Specify the custom directory prefix string used when TFTP_PREFIX is set to 1. For example:- TFTP_PREFIX_STR=tftp_test/
+Default: ""
+Version: stable/pieeprom-2020-01-17.bin  
+
+## Network Boot
+### Server configuration                                                    
+Network boot requires a TFTP and NFS server to be configured.  See [Network boot server tutorial](bootmodes/net_tutorial.md)
+
+Additional notes:-
+* The MAC address on the Pi4 is programmed at manufacture and is not derived from the serial number.
+```                                                                          
+# mac adddress (ip addr) - it should start with DC:A6:32
+ip addr | grep ether | head -n1 | awk '{print $2}' | tr [a-z] [A-Z]                                                                                                                                                
+# serial number                                                                                                                         
+vcgencmd otp_dump | grep 28: | sed s/.*://g
+```
+
+### Installation - firmware update
+Network boot functionality is included in the 2020-02-13 Raspbian Buster release. However, for advanced boot modes it's normally best to use the latest stable software.
+```
+# Update the rpi-eeprom update package                                                                                                 
+sudo apt update
+sudo apt upgrade
+                                                                          
+# Select the stable bootloader release series. The default is 'critical' which is only updated 
+# for major bugs, security fixes or hardware compatibility changes.
+# Network boot is not enabled in the default bootloader.
+# As root:-
+echo FIRMWARE_RELEASE_STATUS="stable" > /etc/default/rpi-eeprom-update
+```
+
+### Enable network boot
+Network boot is not enabled by default in the bootloader. To enable it the bootloader configuration file must be edited.
+```                                                                        
+# Extract the configuration file                                                                                                         
+cp /lib/firmware/raspberrypi/bootloader/stable/pieeprom-2020-01-17.bin pieeprom.bin                                                       
+rpi-eeprom-config pieeprom.bin > bootconf.txt                                                                          
+```
+Change BOOT_ORDER to be 0x21 instead of 0x1. This tells the bootloader to try sd-card boot first and network boot second. You should normally include sd-card (0x1) in the boot sequence in-case of network failure.
+
+### Apply the configuration change to the EEPROM image file                                                                               ```
+rpi-eeprom-config --out pieeprom-netboot.bin --config bootconf.txt pieeprom.bin
+sudo reboot
```

Comment:
Apologies if I'm missing something obvious, but don't the commands shown here just write the config into `/home/pi/pieeprom-netboot.bin`? (and not actually result in the changes getting written to EEPROM?)
Like I said earlier, its always best to be explicit in the documentation about what steps are required... what's obvious to one person might not be to another!